src/h3/mod.rs

```diff
@@ -26,4 +26,830 @@
 
 //! HTTP/3 client and server.
 
+pub mod frame;
```

Comment:
Does this need to be public?

Also, mod declarations should go at the end of the file (see e.g. `src/lib.rs`).